android/src/main/java/com/turboimage/TurboImageViewManager.kt

```diff
@@ -230,6 +230,14 @@ class TurboImageViewManager : SimpleViewManager<TurboImageView>(), LifecycleEven
     }
   }
 
+  @ReactProp(name = "resizeWidth")
+  fun setResizeWidth(view: TurboImageView, resizeWidth: Int?) {
```

Comment:
Maybe a `// todo` here to actually implement this for android - but in the interest of getting this out sooner this is fine.